### ISSUE TITLE
command line arguments

### ISSUE THREAD

**Issue by donald-e-boyce** [HEXRD/hexrdgui#1328]
It would nice for the GUI to have command line arguments, including number of processes.

**Comment by psavery** [comment]
Hi @donald-e-boyce. This is being added in #1339. Here is what it currently looks like:

```
$ hexrdgui -h
usage: hexrdgui [-h] [--ignore-settings] [-n NCPUS]

High energy x-ray diffraction data analysis

options:
  -h, --help            show this help message and exit
  --ignore-settings     Ignore previous settings when HEXRDGUI starts
  -n NCPUS, --ncpus NCPUS
                        Set the number of CPUs to use for parallel operations
```

I tested the `--ncpus` option with the HEDM workflow, and it seemed to work.

**Comment by psavery** [comment]
Fixed in #1339
